config/secrets.js

```diff
@@ -59,3 +62,17 @@ module.exports.updateFetching = function(flag, cb) {
     return cb(err);
   });
 };
+
+// update settings
+module.exports.update = function(type, settings, cb) {
+  cb = cb || function() {};
+
+  for (var key in settings) {
+    var item = type + ':' + key;
+    nconf.set(item, settings[key]);
+  }
+
+  nconf.save(function(err) {
+    return cb(err);
+  });
+};
```

Comment:
It would be better! 
I have focused on the other side of the problem in this pull. 
I put some thought into the DB solution, and I am still undecided about the best solution.
1. A settings collection with key and value fields
2. A simple collection with a single entry containing the whole json

Also there is the need of checking the nconf-mongodb connection. See https://github.com/totherik/nconf-mongodb